src/backends.c

```diff
@@ -395,19 +395,19 @@ void *backends_main(void *ptr) {
 
     info("BACKEND configured ('%s' on '%s' sending '%s' data, every %d seconds, as host '%s', with prefix '%s')", type, destination, source, frequency, hostname, prefix);
 
-    unsigned long long step_ut = frequency * 1000000ULL;
-    unsigned long long random_ut = now_realtime_usec() % (step_ut / 2);
-    time_t before = (time_t)((now_realtime_usec() - step_ut) / 10000000ULL);
```

Comment:
oops! yes it is.
It seems it affects only the first iteration, this is why I haven't found it during testing...